site.config.ts

```diff
@@ -20,6 +20,7 @@ export default siteConfig({
   twitter: 'transitive_bs',
   github: 'transitive-bullshit',
   linkedin: 'fisch2',
+  newsletter: '#', //Use full newsletter URL
```

Comment:
Same comment as the other PR. Let's comment this out so its not enabled by default.

Reply:
Done. Thank you for reviewing!